tests/hash/hash-test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+
+run_success() {
+    RESULT=`./wolfssl $1`
+    if [ $? != 0 ]; then
+        echo "Failed on test \"$1\""
+        exit 99
+    fi
+}
+
+run_fail() {
+    RESULT=`./wolfssl $1`
+    if [ $? == 0 ]; then
+        echo "Failed on test \"$1\""
+        exit 99
+    fi
+}
+
+run_success "-hash sha -in certs/ca-cert.pem -base64enc"
```

Comment:
test depends on the certs directory existing, please sanity check the directory and skip the test if it does not exist.

```
cd wolfclu
./autogen.sh
./configure
make dist
mkdir tmp
cd tmp && tar xvf ../wolfclu-0.0.7.tar.gz
cd wolfclu-0.0.7
./configure && make check


ERROR: tests/hash/hash-test.sh
```